test-alerts.cpp

```diff
@@ -3,6 +3,25 @@
 #include "test/catch.hpp"
 #include "typewise-alert.h"
 
-TEST_CASE("infers the breach according to limits") {
-  REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
+TEST_CASE("infers the temperature hasn't breached any limits according to given cooling type") 
+{
+  REQUIRE(checkAndAlert(TO_CONTROLLER, PASSIVE_COOLING, 12) == NORMAL);
+  REQUIRE(checkAndAlert(TO_EMAIL, HI_ACTIVE_COOLING, 25) == NORMAL);
```

Comment:
These tests will succeed even if Alert was not successful. Why not REQUIRE AlertSuccessful to be true as well?